arkouda/categorical.py

```diff
@@ -11,9 +11,8 @@
 from arkouda.dtypes import resolve_scalar_dtype, str_scalars
 from arkouda.dtypes import int64 as akint64
 from arkouda.sorting import argsort
-from arkouda.pdarraysetops import concatenate, in1d
 from arkouda.logger import getArkoudaLogger
-from arkouda.infoclass import information, pretty_print_information
```

Comment:
@hokiegeek2 why did you remove `pretty_print_information` ?

Reply:
I was wondering the same thing at first but I realized that `Categorical` and `Strings` call `.pretty_print_info` on their components, so the only class that actually needs to import the `infoclass.pretty_print_information` is `pdarrayclass`.

I think I was calling `pretty_print_information` on all the subcomponents at one point before I switched to this delegating down approach and must've forgot to remove the import

Reply:
@mhmerrill cause it's an unused import